### ISSUE TITLE
Provide more verbose output on issues found

### ISSUE THREAD

**Issue by lirantal** [lirantal/lockfile-lint#9]
**Is your feature request related to a problem? Please describe.**
Currently when an issue is found during a linting script run the information provided on the error is pretty limited, such as:

```
ABORTING lockfile lint process due to error exceptions
Invalid URL: false
TypeError [ERR_INVALID_URL]: Invalid URL: false
    at onParseError (internal/url.js:240:17)
    at parse (internal/url.js:249:3)
```

Would be better to provide more verbose information on errors such as the actual package this was detected on and so on.

**Comment by emimuresan** [comment]
Hi @lirantal ! I created a PR for this. Please take a look when you get a chance. Thank you!

**Comment by lirantal** [comment]
Addressed in https://github.com/lirantal/lockfile-lint/pull/13
